packages/pulp/python-django/python-django.spec

```diff
@@ -0,0 +1,69 @@
+# Created by pyp2rpm-3.3.3
+%global pypi_name Django
+%global srcname django
+
+Name:           python-%{srcname}
+Version:        2.2.7
+Release:        1%{?dist}
+Summary:        A high-level Python Web framework that encourages rapid development and clean, pragmatic design
+
+License:        BSD
+URL:            https://www.djangoproject.com/
+Source0:        https://files.pythonhosted.org/packages/source/D/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+
+%description
+Django is a high-level Python Web framework that encourages rapid development
+and clean, pragmatic design. Thanks for checking it out.All documentation is in
+the "docs" directory and online at If you're just getting started, here's how
+we recommend you read the docs:* First, read docs/intro/install.txt for
+instructions on installing Django.* Next, work through the tutorials in
+order...
+
+%package -n     python3-%{srcname}
+Summary:        %{summary}
+%{?python_provide:%python_provide python3-%{srcname}}
+
+Requires:       python3-pytz
+Requires:       python3-setuptools
+Requires:       python3-sqlparse
+%description -n python3-%{srcname}
+Django is a high-level Python Web framework that encourages rapid development
+and clean, pragmatic design. Thanks for checking it out.All documentation is in
+the "docs" directory and online at If you're just getting started, here's how
+we recommend you read the docs:* First, read docs/intro/install.txt for
+instructions on installing Django.* Next, work through the tutorials in
+order...
+
+%prep
+%autosetup -n %{pypi_name}-%{version}
+# Remove bundled egg-info
+rm -rf %{pypi_name}.egg-info
+
+# hard-code python3 in django-admin
+pushd django
+for file in bin/django-admin.py conf/project_template/manage.py-tpl ; do
+    sed -i "s/\/env python/\/python3/" $file ;
+done
+popd
+
+%build
+%py3_build
+
+%install
+%py3_install
+
+%files -n python3-%{srcname}
+%license django/dispatch/license.txt django/contrib/gis/gdal/LICENSE django/contrib/gis/geos/LICENSE django/contrib/admin/static/admin/js/vendor/xregexp/LICENSE.txt django/contrib/admin/static/admin/js/vendor/jquery/LICENSE.txt django/contrib/admin/static/admin/js/vendor/select2/LICENSE.md django/contrib/admin/static/admin/css/vendor/select2/LICENSE-SELECT2.md django/contrib/admin/static/admin/fonts/LICENSE.txt django/contrib/admin/static/admin/img/LICENSE docs/_theme/djangodocs/static/fontawesome/LICENSE.txt LICENSE LICENSE.python
+%doc django/contrib/admin/static/admin/fonts/README.txt django/contrib/admin/static/admin/img/README.txt docs/_theme/djangodocs/static/fontawesome/README.md tests/README.rst README.rst extras/README.TXT
+%{_bindir}/django-admin
+%{_bindir}/django-admin.py
```

Comment:
I'm wondering if we need to clutter $PATH with both

Reply:
Don't *need*, but that's what's generated and I tried not to touch unless really required.